### PR TITLE
flush api for specified cf

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
@BusyJay @siddontang PTAL

### CODE REVIEW

File: librocksdb_sys/src/lib.rs [hunk 1]
```diff
@@ -574,6 +574,10 @@ extern "C" {
     pub fn crocksdb_flush(db: *mut DBInstance,
                           options: *const DBFlushOptions,
                           err: *mut *mut c_char);
+    pub fn crocksdb_flush_cf(db: *mut DBInstance,
+                          options: *const DBFlushOptions,
+                          cf: *mut DBCFHandle,
```
Comment (BusyJay) [inline librocksdb_sys/src/lib.rs]: cf should be the second parameter.

File: src/rocksdb.rs [hunk 2]
```diff
@@ -1772,4 +1780,23 @@ mod test {
...
+
+    #[test]
+    fn flush_cf() {
+        let path = TempDir::new("_rust_rocksdb_flush_cf").expect("");
+        let mut opts = Options::new();
+        opts.create_if_missing(true);
+        let mut db = DB::open(opts, path.path().to_str().unwrap()).unwrap();
+        let opts = Options::new();
+        db.create_cf("cf", &opts).unwrap();
+
```
Comment (hhkbp2) [inline src/rocksdb.rs]: Check `total_sst_files_size` before writing (or flushing) as comparison.

**Comment by hhkbp2** [comment]
Rest LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**